galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,153 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from pulp_ansible.app.models import Collection
+from galaxy_ng.app.models import Namespace
+
+from galaxy_ng.app.api import base as api_base
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request, *args, **kwargs):
+        collection_count = Collection.objects.count()
+        partner_count = Namespace.objects.count()
```

Comment:
I am not sure all namespaces are partners, @awcrosby and @newswangerd can confirm, but maybe we need filtering also here.

Reply:
The current one is using the `v3/namespace/` endpoint with `meta.count` accessor.